src/llamacrawl/services/query/engine.py

```
@@ -47,16 +47,18 @@ def _extract_content_from_payload(payload: dict[str, Any]) -> str:
     # Try to get content from _node_content (LlamaIndex format)
     if "_node_content" in payload and payload["_node_content"]:
```

Ruff (0.14.0)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)